src/client/utils/html.js

```diff
@@ -221,8 +221,17 @@ export function processHtml (html, parentTag, prepareDom) {
         for (let i = 0; i < children.length; i++) {
             const el = children[i];
 
-            if (hasIsNotClosedFlag(el))
+            if (hasIsNotClosedFlag(el)) {
+                if (isScriptElement(el) && nativeMethods.hasAttribute.call(el, 'src')) {
+                    const storedTextContent = el.textContent;
```

Comment:
Need to investigate why do we need to preserve a `textContent`.
Add test for this case if it's need.